tests/v0_tests/test_index.py

```
@@ -40,9 +49,9 @@ def test_create_index_settings_dict(self):
             mock__post = mock.MagicMock()
             @mock.patch("marqo._httprequests.HttpRequests.post", mock__post)
             def run():
-                self.client.create_index(
+                self.create_index(
                     index_name=self.index_name_1,
-                    settings_dict=settings_dict,
+                    index_defaults=settings_dict,
```

the `index_defaults` parameter in `create_index` is misleading because, at least in this case it's actually just the index settings (with the index_defaults being a field within the index_settings dict)

---

the `index_defaults` parameter in `create_index` needs to be more clearly defined.